ecommerce_extensions/payment/urls.py

```diff
@@ -1,12 +1,17 @@
 """Payment processors urls file."""
```

Comment:
I find it weird that we change the file permissions. I will change this locally and squash push the file with the same permissions as the others